src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/Properties/Resources.resx

```diff
@@ -118,14 +118,14 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="PluginTitle" xml:space="preserve">
-    <value>VSCode Workspaces</value>
+    <value>Visual Studio Code Workspaces</value>
   </data>
   <data name="In" xml:space="preserve">
     <value>in</value>
     <comment>Used to indicate the location where something is</comment>
   </data>
   <data name="PluginDescription" xml:space="preserve">
-    <value>Open VSCode previously opened workspaces, remote machines (SSH or Codespaces) and containers.</value>
+    <value>Opens VS Code's previously opened workspaces, remote machines (SSH or Codespaces) and containers.</value>
```

Comment:
```suggestion
    <value>Lists and opens workspaces, remote machines (SSH or Codespaces) and containers previously opened in VS Code.</value>
```

Reply:
@htcfreek 
good suggestion.
Do we need `Lists and...` or can we just says `Opens...` since all modules, in a way or in another, `lists` something.

Reply:
> @htcfreek 
> good suggestion.
> Do we need `Lists and...` or can we just says `Opens...` since all modules, in a way or in another, `lists` something.

@enricogior 
Good point. To open them, I logail neea list. We could use only `opens` or we could use something like `Provides access to ...` which implies list and open too.

Reply:
I will go just with `Opens` as it is consistent with other plugins.